src/app.py

```diff
@@ -161,21 +180,26 @@ async def spawn_process_for_new_agent(object: Object):
     await parent_connection.coro_send(state.scene.serialize())
     state.processes[object.id] = ProcessData(object_id=object.id, process=process, pipe=parent_connection)
 
+async def create_tick():
+    while True:
+        await tick_trigger()
+        await asyncio.sleep(1 - missing_to_full_second())
+
 
 #  __   __       ___  __  __
 # |__) /  \ |  |  |  |__ (__'
 # |  \ \__/ \__/  |  |__ .__)
 
 
 @app.get("/")
-async def get(tick_number: int, background_tasks: BackgroundTasks) -> Dict[str, Union[int, str, List[List[str]]]]:
-    background_tasks.add_task(tick_trigger)
+async def get(tick_number: int) -> Dict[str, Union[int, str, List[List[str]]]]:
     # if player has no scene or player tick number is incorrect
+    state.available_ticks_without_get = 3
     # TODO remove tick number and use scene hash instead e.g. if player_scene_hash not in self.states.keys():
-    if tick_number < 0 or tick_number > state.tick_number:
-        return {"tick_number": state.tick_number, "scene": state.scene.serialize()}
+    if tick_number <= 0 or tick_number > state.tick_number:
```

Comment:
Why was `=` added?

Reply:
Because without it I had strange behaviour on scene host and when I populated map, on one screen I didn't get map.

Reply:
On one screen, as in with one of the players?

Reply:
On one of the player's screen.

Reply:
This seems like a workaround to some race issue, we can't have that.
Every player starts with tick number set to -1, so they should always have the map with `tick_number < 0`

Reply:
When I checked logs, when we don't have any map, and we call `get`, in return we get empty map. so frontend changes tick_number to 0.

Reply:
Oh, interesting. That's a dangerous edge case. Thanks for checking.